### ISSUE TITLE
Bucket location

### ISSUE THREAD

**Issue by maximlamare** [sentinel-hub/byoc-tool#2]
byoc-tool only seems to work with an Amazon bucket located in EU Central (Frankfurt). If specifying a collection linked to a bucket in US-West (Oregon), the tool throws the following error:

```
2020-06-05 17:56:14,352 ERROR: Failed to ingest tiles.
java.lang.RuntimeException: Collection does not exist.
	at com.sinergise.sentinel.byoctool.ingestion.ByocIngestor.ingest(ByocIngestor.java:63)
	at com.sinergise.sentinel.byoctool.cli.IngestCmd.run(IngestCmd.java:157)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1729)
	at picocli.CommandLine.access$900(CommandLine.java:145)
	at picocli.CommandLine$RunAll.handle(CommandLine.java:2157)
	at picocli.CommandLine$RunAll.handle(CommandLine.java:2116)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:1935)
	at picocli.CommandLine.execute(CommandLine.java:1864)
	at com.sinergise.sentinel.byoctool.ByocTool.main(ByocTool.java:112)
```

**Comment by tomislijepcevic** [comment]
Noted. Currently, the tool only accesses the byoc in EU Central deployment.

**Comment by tomislijepcevic** [comment]
Should be fixed in the latest version v0.2.4.
